client/utils/tags.js

```diff
@@ -47,6 +51,49 @@ export const getheadTagComponents = () => {
       name='monetization'
     />
   ];
+  return injectConditionalTags(headTags, homeLocation);
+};
+
+// strips subpath and protocol
+const urlStripper = url => {
+  const noProtocolUrl = url.replace(/(^\w+:|^)\/\//, '');
+  const subpathIndexOrLength =
+    noProtocolUrl.lastIndexOf('/') === -1
+      ? noProtocolUrl.length
+      : noProtocolUrl.lastIndexOf('/');
+  const noSubpathUrl = noProtocolUrl.substr(0, subpathIndexOrLength);
+  return noSubpathUrl;
+};
```

Comment:
```suggestion
```
Do we just want the host?  If so, we can use the URL object.  Since it's a one-liner, I'll suggest the code below.